src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter((person) => person.sex === 'm');
+
+  men = century
+    ? men.filter((person) => Math.ceil(person.died / 100) === century)
+    : men;
+
+  const sumOfAge = men.reduce((prev, man) => (
+    man.died - man.born + prev
+  ), 0);
```

Comment:
You did it 2 times (the same logic) for different arrays
Let's create separate function for this functionality that will accept an array and will return the average people age